Cargo.toml

```diff
@@ -1511,6 +1511,16 @@ description = "Illustrates creating and updating a button"
 category = "UI (User Interface)"
 wasm = true
 
+[[example]]
+name = "flex_display"
+path = "examples/ui/flex_display.rs"
+
+[package.metadata.example.flex_display]
+name = "Flex Display"
+description = "Demonstrates how Display and Visibility work in the UI."
+category = "UI (User Interface)"
+wasm = false
```

Comment:
is there something that wouldn't work in wasm in the example?

Reply:
No, but I don't know much about web dev, so I left it out. 
I see now there's a section in the examples readme about how to build & run the examples for the web. I'll try and test if it works.